src/app/screens/Accounts/NostrSettings/index.tsx

```diff
@@ -47,27 +47,6 @@ function NostrSettings() {
     fetchData();
   }, [fetchData]);
 
-  useEffect(() => {
```

Comment:
Could we keep this code and just not do the toast in the catch block? we could log the error to the console instead